ykman/device.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, descriptor, driver):
         self._descriptor = descriptor
         self._driver = driver
         self.device_name = descriptor.key_type.value
+        self._form_factor = None
```

Comment:
Not sure if it makes sense to differentiate between the case where the formfactor value is missing vs present but 0.